CHANGELOG.rst

```diff
@@ -86,8 +86,14 @@ TBR
   :class:`scrapy.Request <scrapy.http.Request>`'s callback is set to
   :func:`scrapy.http.request.NO_CALLBACK`.
 
+* Fix ``TypeError`` when using Twisted <= 21.7.0 since scrapy-poet was using
+  ``twisted.internet.defer.Deferred[object]`` type annotation before which was
+  not subscriptable.
+
 * Documentation improvements.
 
+* Various test and CI Fixes.
```

Comment:
I think there are also actual bug fixes in this PR

Reply:
Addressed in https://github.com/scrapinghub/scrapy-poet/pull/129/commits/de56e9046a82a646fda44635c221df6e590a8bc1. I've omitted the fixes for the unreleased features like https://github.com/scrapinghub/scrapy-poet/pull/88